app/lib/bk/compat/parsers/circleci/orbs.rb

```diff
@@ -28,6 +19,53 @@ def translator(action, _config)
           ]
         end
       end
+
+      # Docker orb specific implementation
+      class DockerOrb
+        def matcher(orb, _conf)
+          orb.start_with?('docker/')
+        end
+
+        def translator(action, config)
+          method_map = {
+            'docker/build' => -> { translate_docker_build(config) },
+            'docker/install-docker' => -> { validate_and_translate_install_docker(config) }
+          }
+          method_map.fetch(action, -> { ["# Unsupported docker orb action: #{action}"] }).call
+        end
+
+        private
+
+        def validate_and_translate_install_docker(config)
+          install_dir = config.fetch('install-dir', '/usr/local/bin')
+          version = config.fetch('version', 'latest')
+
+          [
+            '# No need for install docker, the agent takes care of that',
```

Comment:
The agent doesn't actually take care of it

```suggestion
            '# Instead of installing docker in a step, we recommend your agent environment to have it pre-installed',
```